corehq/apps/app_manager/templates/app_manager/partials/case_config_shared.html

```diff
@@ -180,3 +182,12 @@
         </a>
     </div>
 </script>
+
+<script type="text/html" id="case-config:refresh-form-questions">
+    <div style="margin-right: 1em;"
```

Comment:
Yeah, style should go in an external file. But for this case, you probably want a [http://getbootstrap.com/components/#btn-groups-toolbar](button toolbar) anyway, not a custom style.